contrib/rootless-cni-infra/rootless-cni-infra

```diff
@@ -2,7 +2,7 @@
 set -eu
 
 ARG0="$0"
-VERSION="0.1.0"
+VERSION=`cat VERSION`
```

Comment:
This requires `VERSION` to be in the image.
If it is complicated, we can just remove `VERSION` and `cmd_entrypoint_version` from the script. (It is not called by Podman currently)